src/builder.rs

```diff
@@ -483,6 +495,29 @@ impl SessionBuilder {
     }
 }
 
+/// Specifies how long the controlling ssh process should stay alive.
+#[derive(Clone, Debug, Default)]
```

Comment:
Let's add `non_exhaustive` so that we can add more variants without breaking anyone:

```suggestion
#[derive(Clone, Debug, Default)]
#[non_exhaustive]
```